src/frontend/src/pages/SettingsPage/SettingsDetails.tsx

```diff
@@ -0,0 +1,160 @@
+    '&.Mui-disabled + .MuiSwitch-track': {
+      opacity: theme.palette.mode === 'light' ? 0.7 : 0.3
+    }
+  },
+  '& .MuiSwitch-thumb': {
+    boxSizing: 'border-box',
+    width: 14,
+    height: 14
+  },
+  '& .MuiSwitch-track': {
+    borderRadius: 18 / 2,
+    backgroundColor: theme.palette.mode === 'light' ? '#E9E9EA' : '#39393D',
+    opacity: 1,
+    transition: theme.transitions.create(['background-color'], {
+      duration: 500
+    })
+  }
+}));
+
+const SettingsDetails: React.FC = () => {
+  const auth = useAuth();
+  const user = useCurrentUser();
+  const [showAlert, setShowAlert] = useState(false);
+  const {
+    isLoading: settingsIsLoading,
+    isError: settingsIsError,
+    error: settingsError,
+    data: userSettingsData
+  } = useSingleUserSettings(user.userId);
+  const {
+    isLoading: secureSettingsIsLoading,
+    isError: secureSettingsIsError,
+    error: secureSettingsError,
+    data: userSecureSettings
+  } = useCurrentUserSecureSettings();
+  const { isLoading: allTeamsIsLoading, isError: allTeamsIsError, data: teams, error: allTeamsError } = useAllTeams();
+
+  if (secureSettingsIsError) return <ErrorPage error={secureSettingsError} message={secureSettingsError.message} />;
+  if (settingsIsError) return <ErrorPage error={settingsError} message={settingsError.message} />;
+  if (allTeamsIsError) return <ErrorPage error={allTeamsError} message={allTeamsError.message} />;
+  if (
+    auth.isLoading ||
+    !auth.user ||
+    settingsIsLoading ||
+    !userSettingsData ||
+    secureSettingsIsLoading ||
+    !userSecureSettings ||
+    allTeamsIsLoading ||
+    !teams
+  )
+    return <LoadingIndicator />;
+
+  const userTeams = teams.filter((team) =>
+    team.members.some((member) => member.userId === user.userId || team.head.userId === user.userId)
+  );
+
+  const logout = () => {
+    setShowAlert(true);
+    setTimeout(() => {
+      auth.signout();
+    }, 2000);
+  };
+
+  return (
+    <PageLayout title="Details">
+      {showAlert && <Alert severity="info">Haha {auth.user?.firstName} bye bye!</Alert>}
+      <Grid container direction="column" spacing={0.5}>
+        <Grid item xs={12} sm={6} md={4} lg={2}>
+          <DetailDisplay label="First Name" content={user.firstName} />
+        </Grid>
+        <Grid item xs={12} sm={6} md={3} lg={2}>
+          <DetailDisplay label="Last Name" content={user.lastName} />
+        </Grid>
+        <Grid item xs={12} sm={7} md={5} lg={3}>
+          <DetailDisplay label="Email" content={user.email} />
+        </Grid>
+        <Grid item xs={12} sm={6} md={4} lg={2}>
+          <DetailDisplay label="Role" content={displayEnum(user.role)} />
+        </Grid>
+        <Grid item xs={12} sm={6} md={4} lg={2}>
+          <DetailDisplay
+            label="Teams"
+            content={userTeams.length === 0 ? 'None' : userTeams.map((team) => team.teamName).join(', ')}
+          />
+        </Grid>
+      </Grid>
+      <PageBlock title={'Organization Settings'}>
```

Comment:
I know this has gone back and forth, but lets put everything that can be changed into the preferences tab. The details tab should just be that readonly list